### ISSUE TITLE
pcb800099 v9

### ISSUE THREAD

**Issue by fu709691151** [linkmarlon/SCHEMATIC-RTD2660H#1]
Very good. Will the firmware for PCB800099 be able to run on this PCB?

**Comment by linkmarlon** [comment]
> Very good. Will the firmware for PCB800099 be able to run on this PCB?

Yes, any firmware from PCB800099 will work on this board.

**Comment by fu709691151** [comment]
Yeah, I'm using RP2040 too, but is there any source code for RTD2660? I can only find some firmware for PCB800099, but I can't modify it

**Comment by Coucouillu2009** [comment]
Hi @linkmarlon and @fu709691151 
I am also interessted in custom firmware RTD2660/2662 and open sources codes. Feel free to share your findings
